project3/index.html

```diff
@@ -0,0 +1,379 @@
+                    style="stroke-width: 2px; fill: rgb(250, 250, 250);" />
+            </svg> </div>
+        <h1> SUPER HERO REGISTRY </h1>
+        <p class="caption"> with great power there must also come-great responsibility</p>
+    </section>
+    <form action="">
+        <section>
+            <h2> IDENTITY </h2>
+
+            <label for="real-name">Real Name</label>
+            <input id="real-name" type="text" placeholder="" required>
+
+            <label for="alias">Alias</label>
+            <input id="alias" type="text" placeholder="" required>
+
+            <label for="email">Email</label>
+            <input id="email" type="email" placeholder="" required>
+        </section>
+        <section class="age">
+            <h2> PERSONAL CHARACTERISTICS </h2>
+            <label for="age">Age</label>
+            <input id="age" type="number" placeholder="" required>
+        </section>
+        <section>
+            <fieldset class="check">
+                <legend>Category</legend>
+                <p>
+                    Please select as many as apply to your origin and how you gain your powers.
+                </p>
+                <div>
+                    <input id="alien" type="checkbox">
+                    <label for="alien">Alien</label>
+                </div>
+                <div>
+                    <input id="cyborg" type="checkbox">
+                    <label for="cyborg">Cyborg</label>
+                </div>
+                <div>
+                    <input id="experiment" type="checkbox">
+                    <label for="experiment">Experiment</label>
+                </div>
+                <div>
+                    <input id="human" type="checkbox">
+                    <label for="human">Human</label>
+                </div>
+                <div>
+                    <input id="inhuman" type="checkbox">
+                    <label for="inhuman">Inhuman</label>
+                </div>
+                <div>
+                    <input id="mutant" type="checkbox">
+                    <label for="mutant">Mutant</label>
+                </div>
+
+            </fieldset>
+
+            <fieldset class="rad">
```

Comment:
It doesn't appear that this classname is used anywhere, so you can just remove it:
```suggestion
            <fieldset>
```